include/seastar/core/shared_ptr_debug_helper.hh

```diff
@@ -25,9 +25,14 @@
 
 #include <thread>
 #include <cassert>
+#include <fmt/format.h>
+
```

Comment:
Why is this include needed?

Reply:
I'm not sure. Maybe the editor added it without me noticing

Reply:
Now I remembered, my original patch printed the differing thread id:s but fmt complained they are not formattable, yet the include directive remained.